src/Sources/SalesForce/Models/SalesForceEntity.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text.Json;
+
+namespace Arcane.Framework.Sources.SalesForce.Models;
+
+/// <summary>
+/// Represents Salesforce entity
+/// </summary>
+public class SalesForceEntity
```

Comment:
The classes that are not intended for use by plugins should always be `internal`, not `public`. This is because the documentation for internal classes is not published as [API docs](https://sneaksanddata.github.io/arcane-docs/arcane-framework-main/README.html)